packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -46,6 +49,19 @@ const memoizedAllConvertChatMessage = memoizeFnAll(
   }
 );
 
+/* @conditional-compile-remove(file-sharing) */
+const extractAttachedFilesMetadata = (metadata: Record<string, string>): FileMetadata[] => {
+  const fileMetadata = metadata['fileSharingMetadata'];
```

Comment:
nit: pull out `'fileSharingMetadata'` to a constant at the top of the file
```
const FILE_SHARING_METADATA_KEY = 'fileSharingMetadata';
```

Reply:
Trying to reduce the number of lines I have to conditionally remove.